packages/node_modules/@ciscospark/react-component-spaces-list/src/index.js

```diff
@@ -2,22 +2,42 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-import SpaceItem from '../space-item';
+import SpaceItem from '@ciscospark/react-component-space-item';
 
 const propTypes = {
-  currentUser: PropTypes.object.isRequired,
+  currentUser: PropTypes.shape({
+    id: PropTypes.string.isRequired
+  }).isRequired,
   formatMessage: PropTypes.func.isRequired,
   hasGroupCalling: PropTypes.bool,
   onCallClick: PropTypes.func,
   onClick: PropTypes.func,
-  spaces: PropTypes.object
+  spaces: PropTypes.oneOfType([
+    PropTypes.arrayOf(
+      PropTypes.shape({
+        avatarUrl: PropTypes.string,
```

Comment:
What do you think about importing the propType for a space from the `spaceItem` component? On one hand, it's less repeated code. On the other, it does somewhat go against our goal of being more explicit in our declarations.